integration-tests/awt fails with latest GraalVM 22.1-dev

### Describe the bug

After the merge of https://github.com/quarkusio/quarkus/pull/20850 integration-tests/awt fails with latest GraalVM 22.0-dev

See https://github.com/oracle/graal/runs/4452553267?check_suite_focus=true#step:8:407

### Expected behavior

Test should pass.

### Actual behavior

Test fails with:

```
org.awaitility.core.ConditionTimeoutException: 
Assertion condition defined as a lambda expression in io.quarkus.awt.it.TestUtil that uses java.nio.file.Path, java.nio.file.Pathjava.util.regex.Pattern, java.util.regex.Patternjava.lang.String Encoders: Log file must not contain blacklisted exceptions. See the offending lines: 
Caused by: java.lang.NullPointerException
Caused by: java.lang.RuntimeException: java.lang.NullPointerException
09:55:48 ERROR [io.qu.ve.ht.ru.QuarkusErrorHandler] (executor-thread-0) HTTP Request to /topng/weird_230.bmp failed, error id: 5c3a314f-8975-4b29-a655-d98354c942e8-4: org.jboss.resteasy.spi.UnhandledException: java.lang.RuntimeException: java.lang.NullPointerException
 in the context of the log: 09:55:44 INFO  [io.qu.aw.it.Application] (main) Available image reader: Standard TIFF image reader
09:55:44 INFO  [io.qu.aw.it.Application] (main) Available image reader: Standard WBMP Image Reader
09:55:44 INFO  [io.qu.aw.it.Application] (main) Available image reader: Standard GIF image reader
09:55:44 INFO  [io.qu.aw.it.Application] (main) Available image reader: Standard PNG image reader
09:55:44 INFO  [io.qu.aw.it.Application] (main) Available image reader: Standard BMP Image Reader
09:55:44 INFO  [io.qu.aw.it.Application] (main) Available image reader: Standard JPEG Image Reader
...
```

### How to Reproduce?

1. Grab latest GraalVM dev build from https://github.com/graalvm/graalvm-ce-dev-builds/releases/
2. `gu install native-image`
2. Set `GRAALVM_HOME` to point to the latest GraalVM
4. `./mvnw -Dnative -Dnative.surefire.skip -pl integration-tests/awt verify`

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/extensions/awt/runtime/src/main/java/io/quarkus/awt/runtime/JDKSubstitutions.java
+++ b/extensions/awt/runtime/src/main/java/io/quarkus/awt/runtime/JDKSubstitutions.java
@@ -1,7 +1,6 @@
 package io.quarkus.awt.runtime;
 
 import java.io.InputStream;
-import java.lang.invoke.MethodHandles;
 import java.util.PropertyResourceBundle;
 
 import com.oracle.svm.core.annotate.Substitute;
@@ -17,12 +16,10 @@ final class Target_com_sun_imageio_plugins_common_I18NImpl {
 
     @Substitute
     private static String getString(String className, String resource_name, String key) {
-        // The property file is now stored in the root of the tree
-        resource_name = "/" + resource_name;
         PropertyResourceBundle bundle = null;
         try {
             // className ignored, there is only one such file in the imageio anyway
-            InputStream stream = MethodHandles.lookup().lookupClass().getResourceAsStream(resource_name);
+            InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream(resource_name);
             bundle = new PropertyResourceBundle(stream);
         } catch (Throwable e) {
             throw new RuntimeException(e);

